common/ManagementTestShared/Track2ManagementRecordedTestBase.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Wasn't the goal to remove common? 

Are we deciding that this is the best thing to do, keep common?